crates/transaction-fuzzer/src/account_universe/universe.rs

```diff
@@ -225,39 +223,52 @@ impl AccountPicker {
 impl AccountPairGen {
     /// Picks two accounts uniformly randomly from this universe and returns mutable references to
     /// them.
-    pub fn pick<'a>(&self, universe: &'a mut AccountUniverse) -> AccountPair<'a> {
-        let [low_idx, high_idx] = universe.picker.pick_pair(&self.pair);
+    pub fn pick<'a>(&self, universe: &'a mut AccountUniverse) -> AccountTriple<'a> {
+        let [low_idx, mid_idx, high_idx] = universe.picker.pick_pair(&self.pair);
         // Need to use `split_at_mut` because you can't have multiple mutable references to items
         // from a single slice at any given time.
         let (head, tail) = universe.accounts.split_at_mut(low_idx + 1);
-        let (low_account, high_account) = (&mut head[low_idx], &mut tail[high_idx - low_idx - 1]);
+        let (mid, tail) = tail.split_at_mut(mid_idx - low_idx);
+        let (low_account, mid_account, high_account) = (
+            head.last_mut().unwrap(),
+            mid.last_mut().unwrap(),
+            tail.last_mut().unwrap(),
+        );
 
         if self.reverse {
-            AccountPair {
+            AccountTriple {
                 idx_1: high_idx,
-                idx_2: low_idx,
+                idx_2: mid_idx,
+                idx_3: low_idx,
                 account_1: high_account,
-                account_2: low_account,
+                account_2: mid_account,
```

Comment:
This way `account_2` is always the `mid_account` so the accounts we pick are no longer uniformly random. Probably not a big deal though.